new-saudi-riyal-for-woocommerce.php

```diff
@@ -79,3 +80,74 @@ function nsrwc_replace_sar_currency_symbol( $currency_symbol, $currency ) {
 }
 
 add_filter( 'woocommerce_currency_symbol', 'nsrwc_replace_sar_currency_symbol', 10, 2 );
+
+/**
+ * Force left position for SAR currency
+ */
+function nsrwc_set_currency_position_left( $position ) {
+    if ( 'SAR' === get_woocommerce_currency() ) {
+        return 'left';
+    }
+    return $position;
+}
+add_filter( 'woocommerce_currency_position', 'nsrwc_set_currency_position_left' );
+
+/**
+ * Enqueue admin CSS if currency is SAR.
+ *
+ * @return void
+ */
+function nsrwc_enqueue_sar_admin_css() {
+    if ( 'SAR' !== get_woocommerce_currency() ) {
+        return;
+    }
+
+    wp_enqueue_style(
+        'sar-admin-style',
+        plugins_url( 'assets/saudi-riyal-font/style.css', __FILE__ ),
+        array(),
+        '1.0'
+    );
+
+    wp_enqueue_script(
+        'sar-admin-blocks-fix',
+        plugins_url( 'assets/js/sar-blocks-fix.js', __FILE__ ),
+        array( 'jquery' ),
+        '1.0',
+        true
+    );
+}
+
+add_action( 'admin_enqueue_scripts', 'nsrwc_enqueue_sar_admin_css' );
+
+/**
+ * Add admin-specific inline CSS for SAR symbol
+ */
+function nsrwc_add_admin_inline_css() {
+    if ( 'SAR' !== get_woocommerce_currency() ) {
+        return;
+    }
+
+    $css = "
+    .woocommerce-Price-currencySymbol,
+    .wc_payment_method .amount,
+    .woocommerce-table--order-details tfoot tr td,
+    .woocommerce-table--order-details tbody tr td {
+        font-family: 'saudi-riyal-font' !important;
+    }
+    ";
+    
+    wp_add_inline_style( 'sar-admin-style', $css );
+}
+add_action( 'admin_enqueue_scripts', 'nsrwc_add_admin_inline_css' );
+
+/**
+ * Apply SAR symbol in admin order list
+ */
+function nsrwc_admin_order_amount_symbol( $formatted_total, $order ) {
+    if ( is_admin() && 'SAR' === get_woocommerce_currency() ) {
+        return '<span class="sar-currency-symbol">' . $formatted_total . '</span>';
+    }
+    return $formatted_total;
+}
+add_filter( 'woocommerce_admin_order_total', 'nsrwc_admin_order_amount_symbol', 10, 2 );
```

Comment:
The store could be multi-currency, so it will have orders in multiple currencies, anyway I can see current plugin code do replace the symbol but the necessary css file was not enqueued, thank to you, this PR fix that.

So there is no need for this as well
```suggestion
```